### PR TITLE
PR #5/5 Astolfo feature/example-update

### PR DESCRIPTION
**Pull request by RealAstolfo**
Changed the example to fit the newer rust api

Co-Authored-By: Thomas ten Cate <ttencate@gmail.com>

### CODE REVIEW

File: examples/dodge-the-creeps/rust/src/player.rs [hunk 1]
```diff
@@ -101,10 +101,10 @@ impl GodotExt for Player {
         }
 
         let change = velocity * delta as f32;
-        let position = self.base.get_global_position().inner() + change;
+        let position = self.base.get_global_position() + change;
         let position = Vector2::new(
-            position.x.max(0.0).min(self.screen_size.inner().x),
-            position.y.max(0.0).min(self.screen_size.inner().y),
+            position.x().clamp(0.0, self.screen_size.x()),
+            position.y().clamp(0.0, self.screen_size.y()),
```
Comment (Bromeon) [inline examples/dodge-the-creeps/rust/src/player.rs:107]: This is probably the only change that's still relevant now.

**Review by Bromeon — COMMENTED** [review]
There's only one possibly relevant change left, do you want to include it or close this PR?

File: examples/dodge-the-creeps/rust/src/player.rs (hunk 1 above)
Comment (RealAstolfo) [reply examples/dodge-the-creeps/rust/src/player.rs]: nice spot.  i merged the suggested change

**Comment by Bromeon** [comment]
Rebased onto `master`, this leaves only the `velocity` changes in the diff.
It's a good way to demonstrate the vector constants though. As such,

bors r+

**Pull request merged**